### PR TITLE
[zh-cn] Updated persistent-volumes.md

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -548,7 +548,7 @@ the following types of volumes:
 * {{< glossary_tooltip text="csi" term_id="csi" >}}
 * flexVolume (deprecated)
 * gcePersistentDisk
-* glusterfs
+* glusterfs (deprecated)
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/storage/persistent-volumes.md:551]: Line 547, 549, and 551 can be localized.

Comment (Sea-n) [reply content/zh-cn/docs/concepts/storage/persistent-volumes.md:551]: Hey @gaogao101, would you like to fix this?

Comment (howieyuen) [reply content/zh-cn/docs/concepts/storage/persistent-volumes.md:551]: @gaogao101 
```suggestion
* glusterfs (已弃用)
```

Comment (gaogao101) [reply content/zh-cn/docs/concepts/storage/persistent-volumes.md:551]: @howieyuen Thank you for your help.

**Comment by tengqm** [comment]
Please resubmit to trigger another CI build.

**Pull request merged**